docs/onnx_custom_op/main.cpp

```diff
@@ -0,0 +1,160 @@
+#include <cstring>
+#include <cassert>
+#include <array>
+
+//! [onnx_custom_op:headers]
+// ie_core.hpp provides definitions for classes need to infer a model (like Core, CNNNetwork, etc.).
+#include <ie_core.hpp>
+// onnx_import/onnx_utils.hpp provides ngraph::onnx_import::register_operator function, that registers operator in ONNX importer's set.
+#include <onnx_import/onnx_utils.hpp>
+// ngraph/opsets/opset5.hpp provides the declaration of predefined nGraph operator set
+#include <ngraph/opsets/opset5.hpp>
+//! [onnx_custom_op:headers]
+
+
+template <size_t N>
+std::ostream& operator<<(std::ostream& os, const std::array<float, N>& array) {
+    for (auto elem : array)
+        os << elem << ", ";
+    return os;
+}
+
+
+int main() {
+//! [onnx_custom_op:model]
+    const std::string model = R"ONNX(
+ir_version: 3
+producer_name: "nGraph ONNX Importer"
+graph {
+  node {
+    input: "in"
+    output: "out"
+    name: "customrelu"
+    op_type: "CustomRelu"
+    domain: "com.example"
+    attribute {
+        name: "alpha"
+        type: FLOAT
+        f: 2
+    }
+    attribute {
+        name: "beta"
+        type: FLOAT
+        f: 3
+    }
+  }
+  name: "custom relu graph"
+  input {
+    name: "in"
+    type {
+      tensor_type {
+        elem_type: 1
+        shape {
+          dim {
+            dim_value: 8
+          }
+        }
+      }
+    }
+  }
+  output {
+    name: "out"
+    type {
+      tensor_type {
+        elem_type: 1
+        shape {
+          dim {
+            dim_value: 8
+          }
+        }
+      }
+    }
+  }
+}
+opset_import {
+  domain: "com.example"
+  version: 1
+}
+)ONNX";
+//! [onnx_custom_op:model]
+
+    // CustomRelu is defined as follows:
+    // x >= 0 => f(x) = x * alpha
+    // x < 0  => f(x) = x * beta
+
+//! [onnx_custom_op:register_operator]
+    ngraph::onnx_import::register_operator(
```

Comment:
What do you demonstrate by this code snippet?

Alternative approach to register custom nGraph operation without creation of IE Extension?

Reply:
> Alternative approach to register custom nGraph operation without creation of IE Extension?
Yes